selfdrive/car/hyundai/values.py

```diff
@@ -88,6 +94,7 @@ class CAR:
   SONATA_LF = "HYUNDAI SONATA 2019"
   TUCSON = "HYUNDAI TUCSON 2019"
   PALISADE = "HYUNDAI PALISADE 2020"
+  PALISADE_2023 = "HYUNDAI PALISADE 2023"
```

Comment:
new generation or no?

Reply:
oh platform code is the same :( this means initial idea for fuzzy FP won't work with these cars just yet

Reply:
in the future, we should be able to combine the two Palisade platforms and detect all the required features (if CAN FD bus, if CAN FD bus and CAN messages, etc), but that will require a bigger chance since we rely on the CAN FD list for static docs generation and likely other places too.

Reply:
Apparently this is a face lifted car, wanna call it that?

> The updated Palisade debuted for the North American market at the [New York International Auto Show](https://en.wikipedia.org/wiki/New_York_International_Auto_Show#2022) on April 13, 2022, for the 2023 model year.[[23]](https://en.wikipedia.org/wiki/Hyundai_Palisade#cite_note-23)[[24]](https://en.wikipedia.org/wiki/Hyundai_Palisade#cite_note-24) It features a revised front end, and an enhanced infotainment system with 12-inch navigation.[[25]](https://en.wikipedia.org/wiki/Hyundai_Palisade#cite_note-25)

https://en.wikipedia.org/wiki/Hyundai_Palisade#Facelift

Reply:
> Apparently this is a face lifted car, wanna call it that?
> 
> > The updated Palisade debuted for the North American market at the [New York International Auto Show](https://en.wikipedia.org/wiki/New_York_International_Auto_Show#2022) on April 13, 2022, for the 2023 model year.[[23]](https://en.wikipedia.org/wiki/Hyundai_Palisade#cite_note-23)[[24]](https://en.wikipedia.org/wiki/Hyundai_Palisade#cite_note-24) It features a revised front end, and an enhanced infotainment system with 12-inch navigation.[[25]](https://en.wikipedia.org/wiki/Hyundai_Palisade#cite_note-25)
> 
> https://en.wikipedia.org/wiki/Hyundai_Palisade#Facelift

@sshane Like this?
`PALISADE_FACE_LIFT = "HYUNDAI PALISADE FACE LIFT"`